mocha.el

```diff
@@ -17,6 +17,7 @@
 ;;; Code:
 (require 'compile)
 (require 'js2-mode)
+(require 'f)
```

Comment:
Ahh, this is just the foundation to the other two PRs (that do need this). Do you want me to rewrite the history or should we let this slip through?